certification_organism/__manifest__.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Certification organism',
+    'version': 'saas~16.4.1.0',
```

Comment:
give a version number like 1.0, this number is not supported for module manifest